packages/client/src/service/txpool.ts

```diff
@@ -362,7 +369,11 @@ export class TxPool {
       add.push({ tx, added, hash })
       this.pool.set(address, add)
       this.handled.set(hash, { address, added })
+
       this.txsInPool++
+      if (this.txGauge !== undefined) {
+        this.txGauge.inc()
+      }
```

Comment:
```suggestion
      this.txGauge?.inc()
```

This syntax works great in ultralight and is a lot code than the `if metrics then metrics.doSomething` as we add more to the system